src/pages/submit.tsx

```diff
@@ -97,21 +97,22 @@ const Submit: NextPage<UserProps> = ({ session }) => {
 									</div>
 
 									<div className='sm:col-span-3'>
-										<label htmlFor='recency' className='block text-sm font-medium text-gray-700'>
+										<label htmlFor='recency-weeks' className='block text-sm font-medium text-gray-700'>
```

Comment:
```suggestion
										<label htmlFor='recency_weeks' className='block text-sm font-medium text-gray-700'>
```
change here also as you have done in the below fields.

Reply:
@debasishbsws thank you but this is an old PR so I'll close it